src/container/rpc.h

```diff
@@ -35,6 +35,9 @@
 	X(CONT_DESTROY,							\
 		0, &CQF_cont_destroy,					\
 		ds_cont_op_handler, NULL),				\
+	X(CONT_DESTROY_BYLABEL,						\
+		0, &CQF_cont_destroy_bylabel,				\
+		ds_cont_op_handler, NULL),				\
```

Comment:
This is a protocol version, you need to bump DAOS_CONT_VERSION on L27 if you change this.  Overall it would help to prepend to this list as well.

Reply:
If you put it here then all the subsequent RPCs will change index so you need a protocol version change, if you put it at the end then you're just adding to the array so technically you don't need one, but in that case you should check for -DER_UNREG on the client.